doc/CMakeLists.txt

```diff
@@ -51,11 +51,7 @@ if(EXISTS ${XELATEX})
   include(cmake/doc-targets.cmake)
 
   # add each of the documents, they have their own CMakeLists.txt files
-  add_subdirectory(acknowledgments)
-  add_subdirectory(auxiliary-programs)
-  add_subdirectory(ems-application-guide)
   add_subdirectory(engineering-reference)
-  add_subdirectory(essentials)
```

Comment:
Remove the CMake add subdirectory commands for those four, since they don't live there anymore.